lambdas/convert-set-to-old-format/src/test/java/uk/gov/di/data/lep/ConvertSetToOldFormatTest.java

```diff
@@ -1027,4 +1046,46 @@ void failingToConvertSetToOldFormatThrowsException() throws JsonProcessingExcept
 
         assertEquals(mappingException, exception.getCause());
     }
+
+    private static Stream<Arguments> sourceSets() {
+        return Stream.of(
+            Arguments.of(deathNotificationSet, "deathNotificationSet"),
+            Arguments.of(deathNotificationSetAliasNameNoMaidenName, "deathNotificationSetAliasNameNoMaidenName"),
+            Arguments.of(deathNotificationSetAliasNameNoMaidenSurname, "deathNotificationSetAliasNameNoMaidenSurname"),
+            Arguments.of(deathNotificationSetEmptyNameDescription, "deathNotificationSetEmptyNameDescription"),
+            Arguments.of(deathNotificationSetNoFamilyName, "deathNotificationSetNoFamilyName"),
+            Arguments.of(deathNotificationSetNoGivenName, "deathNotificationSetNoGivenName"),
+            Arguments.of(deathNotificationSetNoNames, "deathNotificationSetNoNames"),
+            Arguments.of(deathNotificationSetOnlyAliasNameDescription, "deathNotificationSetOnlyAliasNameDescription"),
+            Arguments.of(deathNotificationSetWithUpdateEvent, "deathNotificationSetWithUpdateEvent"),
+            Arguments.of(deathNotificationSetWithYear, "deathNotificationSetWithYear"),
+            Arguments.of(deathNotificationSetWithYearMonth, "deathNotificationSetWithYearMonth")
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("sourceSets")
+    void convertSetApprovalTest(DeathNotificationSet sourceSet, String name) throws JsonProcessingException {
+        var objectMapper = Mapper.objectMapper();
+        var sqsMessage = new SQSMessage();
+
+        sqsMessage.setBody(objectMapper.writeValueAsString(sourceSet));
+        var sqsEvent = new SQSEvent();
+        sqsEvent.setRecords(List.of(sqsMessage));
+
+        var underTest = new ConvertSetToOldFormat(awsService, config, objectMapper);
+
+        when(config.getTargetTopic()).thenReturn("Target Topic");
+
+        underTest.handleRequest(sqsEvent, null);
+
+        ArgumentCaptor<String> captor = ArgumentCaptor.forClass(String.class);
+        verify(awsService).putOnTopic(captor.capture());
+
+        var options = new Options(Scrubbers.scrubAll(
+            new GuidScrubber(),
+            new RegExScrubber("\"iat\":\\d+,", n -> "\"iat\":" + n + ","))
```

Comment:
Just out of curiosity, what's the `n` doing here?

Reply:
Do we need the `iat` at all on this one?

Reply:
The "n" is "how many occurrences matching this pattern has the scrubber seen", so we'll get `"iat":1`, it's clearer for guids, where were get `1` and `2`, which means they're _distinct_ but also deterministic.

Reply:
We possibly don't need it here though, good shout!